Test/comprehensive_performance_test.py

```
@@ -0,0 +1,1115 @@
+"""
+import socket
+import time
+import flatbuffers
+import sys
+import os
+import struct
+import threading
+import random
+import queue
+from concurrent.futures import ThreadPoolExecutor
+import statistics
```

Ruff (0.12.2)

`statistics` imported but unused

Remove unused import: `statistics`

(F401)